src/store/getters/index.ts

```diff
@@ -93,4 +94,8 @@ export default {
   getApplicationStartTime(state: State): Date {
     return state.applicationStartTime;
   },
+
+  getActiveCommunityMembers(state: State): Profile[] {
```

Comment:
@fayeed I think we might want a getter that takes a `communityId` as a param here. And each `CommunityState` could have a `members` field, so we can avoid having `activeCommunityMembers` as a separate we need to keep track of and update. What do you think?

Reply:
Yeah this might be a better approach